packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/delete-encounter-modal.component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, ModalHeader, ModalBody, ModalFooter } from '@carbon/react';
+import styles from './delete-encounter-confirmation-modal.scss';
+
+interface DeleteEncounterConfirmationProps {
+  encounterTypeName?: string;
+  close: Function;
+  onConfirmation: Function;
+}
+
+const DeleteEncounterConfirmation: React.FC<DeleteEncounterConfirmationProps> = ({
+  close,
+  onConfirmation,
+  encounterTypeName,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <ModalHeader closeModal={close} className={styles.productiveHeading03}>
+        {t('deleteEncounter', 'Delete Encounter', { encounter: encounterTypeName })} ?
```

Comment:
We are not using the `encounter` variable anywhere in the text ("Delete Encounter"). Please have a look at the expected string.
Thanks!